bundles/org.eclipse.wst.xml.xpath2.processor/src/org/eclipse/wst/xml/xpath2/processor/StaticNameResolver.java

```diff
@@ -617,6 +617,20 @@ public Void visit(TreatAsExpr taexp) {
 	@Override
 	public Void visit(CastableExpr cexp) {
 		printBinExpr("CASTABLE", cexp);
+
+		SingleType st = (SingleType) cexp.right();
+		QName type = st.type();
+		
+		javax.xml.namespace.QName qName = type.asQName();
+		Function f = _sc.resolveFunction(qName, 1);
+		if (f == null) {
+			if (type.getStringValue().equals("xs:NOTATION") || type.getStringValue().equals("xs:anyAtomicType")) {
```

Comment:
Will the assumption here that the prefix ```xs:``` will always bound to the XSD namespace always hold?  Would it be possible, for example, for us to the the type string of ```xsd:NOTATION```?